app/models/user.rb

```diff
@@ -774,41 +757,23 @@ def self.primer
       readlines.map(&:chomp)
   end
 
-  private_class_method def self.primer_data
-    # How to order - https://stackoverflow.com/a/71282345/3357635
-    users = User.arel_table
-    User.order(arel_function_last_login_if_recent.desc,
-               users[:contribution].desc).
-      limit(1000).pluck(arel_function_login_plus_name).uniq.sort
-  end
+  # NIMMO NOTE: is this method covered by a test?
```

Comment:
It is covered (visited) by a test. 
To see the coverage (without setting a breakpoint and running the tests)
- go to the PR in GitHub
- near the bottom where it shows the CI checks, click Show All Tests
- next to "coverage/coveralls", click Details
- navigate to the file
- go to the method